lib/howl/log.moon

```diff
@@ -45,5 +51,6 @@ error = (message) -> last_error = dispatch 'error', message
 clear = ->
   entries = {}
   last_error = nil
+debug = (msg, ...) -> io.stderr\write '\n'..msg, serpent.block {...}
```

Comment:
Right. I only use it during debugging so the output doesn't bother me. But I'll remove it for now.